hugegraph-llm/src/hugegraph_llm/config/hugegraph_config.py

```diff
@@ -22,8 +22,7 @@
 class HugeGraphConfig(BaseConfig):
     """HugeGraph settings"""
     # graph server config
-    graph_ip: Optional[str] = "127.0.0.1"
-    graph_port: Optional[str] = "8080"
+    graph_url: Optional[str] = "http://127.0.0.1:8080"
```

Comment:
```suggestion
    graph_url: Optional[str] = "127.0.0.1:8080"
```

use non-prefix default value (we already added the info for it)